administrator/components/com_associations/models/associations.php

```diff
@@ -124,6 +124,12 @@ protected function getListQuery()
 			->select($db->quoteName('a.' . $component->fields->ordering, 'ordering'))
 			->from($db->quoteName($component->dbtable, 'a'));
 
+		// Select author for ACL checks
+		if ($component->dbtable != '#__menu')
```

Comment:
why menu? what if other third party components doesn't sipport checkout?
Isn't it better ...
`!is_null($component->fields->created_by)`